tests/phpunit/integration/Core/User_Surveys/REST_User_Surveys_ControllerTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Class Google\Site_Kit\Tests\Core\User_Surveys\REST_User_Surveys_ControllerTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\User_Surveys
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\User_Surveys;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\User_Surveys\REST_User_Surveys_Controller;
+use Google\Site_Kit\Tests\TestCase;
+
```

Comment:
Let's add a `User_Surveys` group similar to other core test cases.

Reply:
Not sure what you mean :thinking: Which group do I need to add and where?

Reply:
Ah sorry, I was referring to a PHPUnit group annotation: `@group User_Surveys` to the test's doc block.

Looking again it's probably not necessary right now since there probably wouldn't be anything else to go in it, so let's not worry about it for now.